lib/vquic/quiche.c

```diff
@@ -404,10 +406,16 @@ static CURLcode flush_egress(struct connectdata *conn, int sockfd,
       return CURLE_SEND_ERROR;
     }
 
-    sent = send(sockfd, out, sent, 0);
-    if(sent < 0) {
-      failf(conn->data, "send() returned %zd\n", sent);
-      return CURLE_SEND_ERROR;
+resend:
+    socksent = send(sockfd, out, sent, 0);
+    if(socksent < 0) {
+      if(errno == EAGAIN || errno == EWOULDBLOCK || errno == EINTR) {
+        goto resend;
```

Comment:
this is a busy-loop... shouldn't this rather wait for the socket to become writable?

This also makes the function annoyingly blocking, which we should fix as well but could of course be done separately.

Reply:
OK, I have done as what you said in my new commit.